examples/simple/simple_test.go

```diff
@@ -24,19 +24,38 @@ import (
 	eventstore "github.com/looplab/eventhorizon/eventstore/memory"
 	readrepository "github.com/looplab/eventhorizon/readrepository/memory"
 
+	"testing"
+
 	"github.com/looplab/eventhorizon/examples/domain"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
```

Comment:
I would prefer to only use the standard lib for testing. I once used a third party testing lib (maybe it was testify) but have since rewritten all tests without it. It makes the tests harder to read (for those unfamiliar with the test lib) and adds an unnecessary dependency to maintain.